### ISSUE TITLE
[IOS] Can no longer debug

### ISSUE THREAD

**Issue by PedroCigaw** [dotnet/maui#4297]
### Description

When targetting IOS and setting a break point the break point gets hit but I cannot see any code.

<img width="635" alt="ios_debug" src="https://user-images.githubusercontent.com/65864520/150876856-25e6f5cc-dd34-4e07-b111-39ea1b7a04ea.png">

This was on a fresh windows 11 install with visual studio 2022 17.1 preview 3.  Just to confirm I tried this on two machines and two different iphones.  Debugging a Xamarin app works fine.

### Steps to Reproduce

1. Install Maui App
2. Set break point anywhere
3. Debug to local device

### Version with bug

Preview 12 (current)

### Last version that worked well

Preview 11

### Affected platforms

iOS

### Affected platform versions

iOS 15

### Did you find any workaround?

No but this makes life very difficult  any suggestions?

**Comment by drasticactions** [comment]
If you create a new iOS app without MAUI and try to break inside it, does it work? My guess would be that it's more of A iOS SDK issue than anything MAUI specific.

**Comment by PedroCigaw** [comment]
I have now tried this on VS 17.4 and 17.5 using Maui preview 12 but still have the same issue.  

@drasticactions  not sure what you mean by new iOS app without Maui?  If I create a new Xamarin Forms app I can debug with not problems, but no luck on when it comes to Maui.

I have tried clearing all remnants of Visual Studio and Xamarin from pc.  Installed new version of VS Preview 17 tried both enterprise and community editions.

Steps I followed from new test machine setup.  
1. Install windows 11 - 21H2 22000.469
2. Install ITunes - Sync phone.
3. Install Visual Studio Preview
4. Create Maui app and try debug.  Fails!
5. Create Xamarin Forms.  Success!

They only difference I can see from Xamarin Forms app to Maui is I can pair to a Mac on Xamarin Project, but cannot for Maui as it fails to install the dotnet sdk, but given I am trying to debug via local device connected to the pc I wouldn't have thought that would make a difference, also debugging was working fine on Preview 11.
